src/test/java/spec/concordion/common/command/example/FailFastFixture.java

```diff
@@ -0,0 +1,16 @@
+package spec.concordion.common.command.example;
+
+import org.concordion.api.ExpectedToFail;
+import org.concordion.api.FailFast;
+import org.concordion.integration.junit4.ConcordionRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(ConcordionRunner.class)
+@FailFast
+@ExpectedToFail
```

Comment:
With `@ExpectedToFail` present, the specification does not exhibit the failure that is fixed in `RunCommand`.

You will need to remove `@ExpectedToFail` and then also move this fixture and related specification from the `src/test` to the `src/test-dummies` folder. This will make sure that this failing specification is excluded from the results.

I have also noticed that `SpecCallingFailFastFixture` is not linked to the specification suite (under `Concordion.html`). I'm happy to address this in a separate PR.